src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeDefinition.xml

```diff
@@ -70,7 +70,7 @@
     <Child Name="Security"/>
   </Node>
 
-  <Node Name="Tables" LocLabel="SR.SchemaHierarchy_Tables" BaseClass="ModelBased" Strategy="MultipleElementsOfType" ChildQuerierTypes="SqlTable" TreeNode="TableTreeNode">
+  <Node Name="Tables" LocLabel="SR.SchemaHierarchy_Tables" BaseClass="ModelBased" Strategy="MultipleElementsOfType" ChildQuerierTypes="SqlTable" TreeNode="TableTreeNode" ValidFor="NotSqlDemand">
```

Comment:
>ValidFor [](start = 170, length = 8)

Hrm. If this was the first I'd say I'd rather see a NotValidFor property instead. But since it isn't I'll leave it up to you whether you want to do the extra work - feel free to leave it as is. Having to create separate Not* flags for each combination of "Not" is going to become annoying as we add more types though.

Reply:
Definitely it would be useful. Unfortunately I don't believe that I have bandwidth for this right away, but we can sync about this.